content/en/platform/corda/4.8/enterprise/operations/deployment/generating-a-node.md

```diff
@@ -227,7 +227,7 @@ rpcUsers = [[ user: "user1", "password": "test", "permissions": ["StartFlow.net.
 
 
 * `configFile` <string>
-    * For extending configuration of nodes. (see [extended node configuration](#generating-a-node-extended-config))
+    * For extending configuration of nodes. (see [extended node configuration](#optional-configuration))
```

Comment:
Should this link here, since it's in the same section? Maybe another file, like: https://docs.r3.com/en/platform/corda/4.5/enterprise/node/setup/corda-configuration-file.html , has more info? I don't know this part of the docs that well, so trusting your judgement.